### ISSUE TITLE
Computing Perplexity?

### ISSUE THREAD

**Issue by Yueeeeeeee** [cse253-team-repo/PA4#1]
Is it correct to use torch.exp(loss) to compute perplexity?

**Comment by Yueeeeeeee** [comment]
Correct
